bitnami/jenkins/values.yaml

```diff
@@ -23,127 +63,201 @@ image:
   ## Optionally specify an array of imagePullSecrets.
   ## Secrets must be manually created in the namespace.
   ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
-  ## Example:
+  ## e.g:
   ## pullSecrets:
   ##   - myRegistryKeySecretName
   ##
   pullSecrets: []
+  ## Enable debug mode
+  ##
+  debug: false
 
-## Force target Kubernetes version (using Helm capabilites if not set)
-##
-kubeVersion:
-
-## String to partially override common.names.fullname template (will maintain the release name)
-##
-# nameOverride:
-
-## String to fully override common.names.fullname template
-##
-# fullnameOverride:
-
-## Deployment pod host aliases
-## https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/
-##
-hostAliases: []
-
-## Add labels to all the deployed resources
-##
-commonLabels: {}
-
-## Add annotations to all the deployed resources
-##
-commonAnnotations: {}
-
-## Kubernetes Cluster Domain
-##
-clusterDomain: cluster.local
-
-## Extra objects to deploy (value evaluated as a template)
-##
-extraDeploy: []
-
-## User of the application
+## @section Jenkins Configuration parameters
+## Jenkins settings based on environment variables
 ## ref: https://github.com/bitnami/bitnami-docker-jenkins#configuration
+
+## @param jenkinsUser Jenkins username
 ##
 jenkinsUser: user
-
-## Application password
+## @param jenkinsPassword Jenkins user password
 ## Defaults to a random 10-character alphanumeric string if not set
-## ref: https://github.com/bitnami/bitnami-docker-jenkins#configuration
 ##
-# jenkinsPassword:
-
-## Jenkins home directory
+jenkinsPassword: ""
+## @param jenkinsHost Jenkins host to create application URLs
 ##
-jenkinsHome: /opt/bitnami/jenkins/jenkins_home
-
-## Allows to disable the initial Bitnami configuration for Jenkins
+jenkinsHost:
```

Comment:
I don't see this parameter being used in the templates

Reply:
Found it, but please explain the reason for such a complex method

Reply:
I will simplify it a little bit, I just copy-paste what we have for other apps such as Magento